Lombiq.Tests.UI/Services/OrchardCoreUITestExecutorConfiguration.cs

```diff
@@ -186,6 +186,13 @@ public Func<LogEntry, bool> BrowserLogFilter
     /// </summary>
     public bool UseAzureBlobStorage { get; set; }
 
+    /// <summary>
+    /// Gets or sets a value indicating whether Elasticsearch will be used. When enabled, the test will assign a unique
+    /// prefix to the test, ensure that Elasticserach caches are flushed before starting the test, and clean up the
```

Comment:
Shouldn't this say: "the test will assign a unique prefix to the index", instead of "the test will assign a unique prefix to the test"?

Reply:
Sure.
```suggestion
    /// Gets or sets a value indicating whether Elasticsearch will be used. When enabled, the test will assign a unique
    /// prefix to the INDEX, ensure that Elasticserach caches are flushed before starting the test, and clean up the
```